api/HTMLElement.json

```diff
@@ -1136,7 +1140,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "4"
```

Comment:
"hidden" is also a reflected attribute and listed in global_attributes.json. I will revert this and other changes like it, as it's possible the content attribute was supported before the reflected attribute, but when that's the case we might need to add notes.